src/components/Button.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styles from "./Button.module.css";
+
+const ButtonComponent = (props) => {
+  return (
+    <>
+      <button
```

Comment:
you can add onClick here and pass the function you want to execute as a prop such as `<button onClick={props.handleClick}>{props.text}</button>`